### PR TITLE
Replace codimd with hedgedoc

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Everything

### Description
This PR replaces the last occurrences of "codimd" with "hedgedoc"

### Steps

- [x] Added implementation
- [x] Added / updated tests
- [x] Added / updated documentation
- [x] Extended changelog
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ SPDX-License-Identifier: CC-BY-SA-4.0
     - `{%gist user/12345 %}` -> https://gist.github.com/user/12345
     - `{%slideshare user/my-awesome-presentation %}` -> Embedding removed
     - `{%speakerdeck foobar %}` -> Embedding removed
-- The use of `sequence` as code block language ([Why?](https://community.codimd.org/t/frequently-asked-questions/190))
```
Comment (ErikMichelson) [inline CHANGELOG.md:18]: I would use the FAQ on the webpage instead.

File: CHANGELOG.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 - The `[name=...]`, `[time=...]` and `[color=...]` tags may now be used anywhere in the document and not just inside of blockquotes and lists.
 - The <i class="fa fa-picture-o"/> (add image) and <i class="fa fa-link"/> (add link) toolbar buttons put selected links directly in the `()` instead of the `[]` part of the generated markdown.
 - The help dialog has multiple tabs, and is a bit more organized.
-- Use KaTeX instead of MathJax. ([Why?](https://community.codimd.org/t/frequently-asked-questions/190))
+- Use KaTeX instead of MathJax. ([Why?](https://community.hedgedoc.org/t/frequently-asked-questions/190))
```
Comment (ErikMichelson) [inline CHANGELOG.md:71]: FAQ on webpage

File: README.md [hunk 3]
```diff
@@ -18,11 +18,11 @@ You'll need at least Node 12. We use [yarn](https://yarnpkg.com/) for our depend
 
 ## Development mode
 
-1. Clone this repo (e.g. `git clone https://github.com/codimd/react-client.git hedgedoc-react-client`)
+1. Clone this repo (e.g. `git clone https://github.com/hedgedoc/react-client.git hedgedoc-react-client`)
 2. Go inside the repo (e.g. `cd hedgedoc-react-client`)
 3. Run `yarn install`
 4. Either run
-    - `yarn start:dev` (expects [a server](https://github.com/codimd/server/tree/develop) running under [http://localhost:3000](http://localhost:3000))
+    - `yarn start:dev` (expects [a server](https://github.com/hedgedoc/server/tree/develop) running under [http://localhost:3000](http://localhost:3000))
```
Comment (ErikMichelson) [inline README.md:25]: It's `hedgedoc/hedgedoc` and not `server` anymore.

File: src/links.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
     "chat": "https://matrix.to/#/#hedgedoc:matrix.org",
-    "community": "https://community.codimd.org",
-    "faq": "https://community.codimd.org/t/frequently-asked-questions/190",
-    "githubOrg": "https://github.com/codimd/",
-    "mastodon": "https://social.codimd.org",
-    "translate": "https://translate.codimd.org",
-    "webpage": "https://codimd.org"
+    "community": "https://community.hedgedoc.org",
+    "faq": "https://community.hedgedoc.org/t/frequently-asked-questions/190",
```
Comment (ErikMichelson) [inline src/links.json:4]: FAQ on webpage

**Review by ErikMichelson — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mrdrogdrog) [reply CHANGELOG.md:18]: done

File: CHANGELOG.md (hunk 2 above)
Comment (mrdrogdrog) [reply CHANGELOG.md]: done

File: README.md (hunk 3 above)
Comment (mrdrogdrog) [reply README.md]: don

File: src/links.json (hunk 4 above)
Comment (mrdrogdrog) [reply src/links.json]: done

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Pull request merged**